src/components/RequesterPaysModal.js

```diff
@@ -21,21 +20,10 @@ const requesterPaysHelpInfo = div({ style: { paddingTop: '1rem' } }, [
 ])
 
 const RequesterPaysModal = ({ onDismiss, onSuccess }) => {
-  const [loading, setLoading] = useState(false)
-  const [workspaceList, setWorkspaceList] = useState([])
-  const [selectedGoogleProject, setSelectedGoogleProject] = useState(requesterPaysProjectStore.get())
-  const signal = useCancellation()
+  const { workspaces, loading } = useWorkspaces()
+  const billableWorkspaces = _.filter(workspace => isGoogleWorkspace(workspace) && (workspace.accessLevel === 'OWNER' || workspace.accessLevel === 'PROJECT_OWNER'), workspaces)
```

Comment:
How do we know that workspaces will be finished loading at the time when this filter is called?

Reply:
They may not be. The `workspaces` returned by `useWorkspaces` is either undefined or an array of workspaces. Lodash's filter handles undefined and returns an empty array in that case.